platforms/jvm/plugins-jvm-test-suite/src/integTest/groovy/org/gradle/testing/testsuites/TestSuitesIntegrationTest.groovy

```diff
@@ -485,50 +485,6 @@ class TestSuitesIntegrationTest extends AbstractIntegrationSpec {
         result.assertTaskExecuted(":integTest")
     }
 
-    @Issue("https://github.com/gradle/gradle/issues/18622")
```

Comment:
Instead of deleting this test does it make sense to verify the new behavior of the test task without the conventions?

Reply:
I considered that, but thought it would be hard to test.

I just checked and you get a terrible error, so I've updated this slightly to still provide a convention, but that convention is an empty collection.

This matches what we would have if we were using `ConfigurableFileCollection`s and not convention mapping.

Reply:
Is there a solution we can use that doesn't involve convention mapping or even requiring applying a plugin at all?

Can we just update Test to be resilient when the classpath is null?

Reply:
We have to use convention mapping until this property is a CFC at least, if we want to provide a convention.

It's not new that Test tasks are not usable without applying a plugin and I think we want to make it so you have to apply plugins to get access to built-in task types (at some point), so I wouldn't go out of our way to make Test more special.

Reply:
I would like to discuss this further, I don't think the convention mapping solution is the right approach

Reply:
ptal